Userland/Applications/Piano/RollWidget.h

```diff
@@ -61,4 +62,10 @@ class RollWidget final : public GUI::ScrollableWidget {
     Optional<Gfx::IntPoint> m_note_drag_start;
     Optional<RollNote> m_note_drag_location;
     int m_drag_note;
+
+    RefPtr<Gfx::Bitmap> m_background;
+    int m_prev_zoom_level { m_zoom_level };
+    int m_prev_scroll_x { horizontal_scrollbar().value() };
+    int m_prev_scroll_y { vertical_scrollbar().value() };
+    bool viewport_changed();
```

Comment:
`viewport_changed` can be `const`. It should also reside with the other methods.